bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/providers/ConfigUtils.java

```diff
@@ -85,7 +85,7 @@ public static boolean containsUnsupportedMembers(BUnionType unionType) {
     }
 
     private static boolean isSimpleSequenceType(int tag) {
```

Comment:
```suggestion
    private static boolean isPrimitiveOrSequenceType(int tag) {
```

Reply:
I missed it. Thank you for pointing that out. Fixed with 2fa8603fa532bca9f27e348eef23ea57ad99a88d